docs/ManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -17,6 +17,10 @@ In this guide, we will create the following components.
 * A Bastion instance with a public IP.
 * Azure container insights.
 
+## Prerequisites
+
+You need to have enough permissions to deploy Scalar DL on Azure. Please see [Cloud permissions document](./CloudPermissions.md) for more details.
```

Comment:
Ditto. We need to explain the permissions are required for what purposes clearly.

Reply:
Modified sentence specified why it is needed in 6239a04